apps/frontend/src/pages/onboarding/onboarding.tsx

```diff
@@ -133,20 +133,24 @@ const Onboarding: React.FC = () => {
 							})}
 							<div className={styles["button-container"]}>
 								{!isLastQuestion && (
+									<div className={styles["button-wrapper"]}>
+										<Button
+											isFluid
+											label="BACK"
```

Comment:
Why is Back button only displayed when it is not a last question? It should be displayed when it is not the first question